etl/steps/grapher/un_sdg/2022-07-07/output/variable_replacements.json

```diff
@@ -1,297 +1,155 @@
-    "415850": "430098",
-    "415851": "430101",
-    "415852": "430097",
-    "415853": "430099",
-    "415854": "430094",
-    "415855": "430100",
-    "415856": "430103",
-    "415857": "430093",
-    "415858": "430102",
-    "415859": "430096",
-    "415860": "430095",
-    "415879": "429910",
-    "415887": "431095",
-    "415889": "431299",
-    "415895": "428010",
-    "415896": "428011",
-    "416109": "430628",
-    "416110": "430248",
-    "416111": "430249",
-    "416113": "430317",
-    "416114": "430316",
-    "416115": "430318",
-    "416122": "428066",
-    "416123": "428067",
-    "416124": "428065",
-    "416125": "428068",
-    "416126": "428957",
-    "416150": "428036",
-    "416151": "428037",
-    "416152": "428038",
-    "416154": "428980",
-    "416155": "428983",
-    "416156": "428984",
-    "416157": "428982",
-    "416158": "428979",
-    "416159": "428981",
-    "416169": "428082",
-    "416194": "428932",
-    "416195": "428008",
-    "416196": "429335",
-    "416217": "428028",
-    "416221": "428020",
-    "416228": "429322",
-    "416229": "429324",
-    "416230": "427939",
-    "416231": "427940",
-    "416232": "427941",
-    "416233": "427942",
-    "416234": "427944",
-    "416235": "427937",
-    "416237": "429007",
-    "416243": "429323",
-    "416244": "429329",
-    "416246": "428964",
-    "416247": "428963",
-    "416249": "428966",
-    "416251": "429002",
-    "416252": "429000",
-    "416268": "427984",
-    "416269": "427985",
-    "416340": "431115",
-    "416345": "431316",
-    "416356": "430077",
-    "416357": "430080",
-    "416368": "427992",
-    "416369": "427990",
-    "416384": "431325",
-    "416386": "431312",
-    "416391": "431321",
-    "416410": "428190",
-    "416640": "429423",
-    "416641": "430078",
-    "416644": "430086",
-    "416646": "430116",
-    "416647": "430118",
-    "416648": "430115",
-    "416649": "430117",
-    "416650": "431406",
-    "416655": "431109",
-    "416949": "429375",
-    "416954": "430242",
-    "416958": "430284",
-    "416981": "429388",
-    "416982": "429390",
-    "416983": "429389",
-    "416984": "430081",
-    "416989": "431776",
-    "416990": "431777",
-    "416996": "431782",
-    "416997": "431783",
-    "416998": "431404",
-    "417004": "431330",
-    "417005": "431331",
-    "417006": "431357",
-    "417007": "431333",
-    "417026": "431409",
-    "417027": "431407",
-    "417028": "431408",
-    "417029": "430092",
-    "417033": "431416",
-    "417034": "431417",
-    "417037": "431410",
-    "417043": "431364",
-    "417044": "431362",
-    "417045": "431359",
-    "417046": "431361",
-    "417067": "431399",
-    "417068": "431390",
-    "417071": "431396",
-    "417076": "431393",
-    "417077": "431304",
-    "417079": "429400",
-    "417082": "429370",
-    "417100": "430041",
-    "417159": "430042",
-    "417272": "430121",
-    "417273": "430122",
-    "417421": "431708"
-}
+    "414079": "456880",
```

Comment:
(The list got shorter, just checking whether this is expected)

Reply:
Ah yeh I think that's okay - the variable matching and chart revisions is still a manual step. I wonder if we should add it as a dag step or if that is overkill, I don't think we would want it run every time but maybe we don't want manual bits of code hanging around in the etl?

Reply:
Hmm, interesting idea. Maybe not exactly a dag step, but some helper command would be helpful.

I really like your `match_vars_crs.py` script for simplicity though. If we decide to automate it, I'd just steal it anyway.

Reply:
Yeh maybe an optional flag would be cool! okay will merge this now :)